firehose/firehose.go

```diff
@@ -228,7 +228,8 @@ func (output *OutputPlugin) processRecord(record map[interface{}]interface{}) ([
 	data = append(data, []byte("\n")...)
 
 	if len(data) > maximumRecordSize {
-		return nil, fmt.Errorf("Log record greater than max size allowed by Kinesis")
+		data = data[:maximumRecordSize]
+		logrus.Warnf("The record %v is truncated since it is larger than 1000Kib.", record)
```

Comment:
We don't want to print any portion of the record, since it is large...

How about just:

```
logrus.Warnf("Found record with %d bytes, truncating to 1000Kib", len(data));
data = data[:maximumRecordSize]
```

Reply:
Yeah..make sense